cmd/config/init/flags.go

```diff
@@ -19,6 +21,10 @@ func addFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP(FlagNames.HubID, "", StagingHubID, fmt.Sprintf("The ID of the Dymension hub. %s", getAvailableHubsMessage()))
 	cmd.Flags().StringP(FlagNames.RollappBinary, "", "", "The rollapp binary. Should be passed only if you built a custom rollapp")
 	cmd.Flags().StringP(FlagNames.TokenSupply, "", defaultTokenSupply, "The total token supply of the RollApp")
+	cmd.Flags().UintP(FlagNames.Decimals, "", 18,
```

Comment:
IMO this flag should be hidden for the default case
It's just confusing, and we anyway expect it to be 18